nucliadb_sdk/nucliadb_sdk/tests/test_search.py

```diff
@@ -292,6 +292,7 @@ def test_search_resource(kb: KnowledgeBoxObj, sdk: nucliadb_sdk.NucliaDB):
         min_score=0.70,
         page_number=0,
         page_size=20,
+        with_duplicates=True,
```

Comment:
I don't get why this was passing before, but now we need this set to `True`. Have you fixed a bug on the node?

Reply:
yes, there was a bug that was returning duplicates in some cases.